strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/search/MavenIndexerSearchProviderTest.java

```diff
@@ -32,6 +32,7 @@
 @ActiveProfiles(profiles = "test")
 @ContextConfiguration(classes = Maven2LayoutProviderTestConfig.class)
 @Execution(CONCURRENT)
+@Disabled
```

Comment:
@carlspring why we need to disable this?

Reply:
We don't need to. There was a problem at the time and I had to temporarily disable it. Thanks for bringing this up and reminding me! :)

Reply:
rolled back